src/libraries/System.Runtime.InteropServices.JavaScript/src/Runtime.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    public static partial class Runtime
+    { }
```

Comment:
What's the purpose of this type?

Reply:
So that a library is actually created.  Did not like it myself but without it the project does not build because the interopservices library is all internal static.

Reply:
Do you plan to reorganize this PR so that this library has public methods and it is used by the HttpClient ?

If not, I may be best to drop the System.Runtime.InteropServices.JavaScript library from this PR since it is just a small unused skeleton that we can add anytime in future.

Reply:
Jan right now the plan was to keep this as internal for running tests against.  We will discuss this in our weekly reunion to see the viability of this.

Reply:
Added this library and referenced it from System.Net.Http project